External/sollve_vv/CMakeLists.txt

```diff
@@ -3,14 +3,32 @@
 
 include(External)
 
-option(TEST_SUITE_SOLLVEVV_FORCE_ALL "Execute all SOLLVE V&V tests, even those known to be unsupported by Clang" OFF)
+option(TEST_SUITE_FORCE_ALL "Execute all SOLLVE V&V tests, even those known to be unsupported by Clang" OFF)
```

Comment:
Can we put the SOLLVEVV back? This option gives the user to modify for the SOLLVEVV - we might want to keep that different from the rest of them